### PR TITLE
added viscous damper to elasticity to make firmoviscous material

### PR DESCRIPTION
**Pull request by bobmyhill**
This PR adds a viscous damper in isostrain with the elastic element in elasticity.cc to create a firmoviscous model. This is one way to create a stabilized elastic rheology that can be incorporated into composite models without changing the elastic timestep (see #3770 for the timestep method, now merged).

* [x] I have followed the [instructions for indenting my code](../blob/master/CONTRIBUTING.md#making-aspect-better).
* [x] I have tested my new feature locally to ensure it is correct.
* [x] I have [created a testcase](http://www.math.clemson.edu/~heister/manual.pdf#sec%3Awriting_tests) for the new feature/benchmark in the [tests/](../blob/master/tests/) directory.
* [x] I have added a changelog entry in the [doc/modules/changes](../blob/master/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

File: tests/damped_viscoelastic_bending_beam.prm [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# This test checks whether the viscoelastic bending beam benchmark
+# run successfully. In particular, this benchmark is a good test to
```
Comment (naliboff) [inline tests/damped_viscoelastic_bending_beam.prm:2]:
```suggestion
# run successfully in combination with a viscous damper for the elastic viscosity. In particular, this benchmark is a good test to
```

File: tests/damped_viscoelastic_bending_beam.prm [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+# ensure that a model with compositional fields that affect material
+# properties works.
```
Comment (naliboff) [inline tests/damped_viscoelastic_bending_beam.prm:4]: For simplicity, I think you could remove the two sentences after the sentence that contains "run successfully".

File: tests/damped_viscoelastic_bending_beam.prm [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  subsection Box
+    set X repetitions = 15
+    set Y repetitions = 10
+  end
+end
+
+# Material model, adding viscous dampers to each material.
+# The material response is only significantly affected if
+# the damper viscosity is similar to or greater than
+# the product of the elastic timestep and the shear modulus.
```
Comment (naliboff) [inline tests/damped_viscoelastic_bending_beam.prm:22]:
```suggestion
# the product of the elastic timestep and the shear modulus.
# A damper value of 0 Pas corresponds to purely elastic deformation.
```

**Review by naliboff — CHANGES_REQUESTED** [review]
@bobmyhill - fantastic, thanks for adding! Only a few minor comments about documentation. Otherwise, looks good to go.

**Comment by bobmyhill** [comment]
Great, thanks @naliboff!

**Review by naliboff — APPROVED** [review]

**Pull request merged**